src/calendar/components/CalendarCell.tsx

```diff
@@ -36,7 +38,12 @@ export default function CalendarCell({
 
   return (
     <td
-      style={{ verticalAlign: 'top', borderTop: '1px solid', height: 64 }}
+      style={{
```

Comment:
Is there a reason for not using a stylesheet file?

Reply:
What do you mean?

Reply:
like a CSS file, with styling.

Reply:
oh lol read it as 'spreadsheet file'

I usually use inline styles whenever I don't need to reuse a style, but sure if I need to reuse a css class, I'll make a css file.

Reply:
just a thought